checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -23,6 +23,7 @@ class CustomRegexDetector(RegexBasedDetector):
     secret_type = "Regex Detector"  # noqa: CCE003 # nosec
     denylist: Set[Pattern[str]] = set()  # noqa: CCE003
     MAX_FILE_SIZE: int = 4 * 1024
+    MAX_LINE_LENGTH: int = 10_000
```

Comment:
I had no idea python supports this syntax (rather than 10000)

Reply:
yeah, some of the small hidden gems 😄